config/metrics/network_policy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-from-openshift-user-workload-monitoring
+  namespace: etcd-shield
+spec:
+  podSelector:
+    matchLabels:
+      apps: etcd-shield
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: openshift-user-workload-monitoring
```

Comment:
AIUI, this is openshift only. Should we avoid it here and move it to infra-deployments?

Reply:
Yeah, FWIW the deployment manifests in this repository are definitely openshift-only for now (see certificate management, for one).  They're here more for assisting in testing purposes more than anything else.